docs/source/setup/concepts.rst

```diff
@@ -0,0 +1,43 @@
+Concepts
+========
+
+This page explains how different attributes of PyVRP's data model relate.
+
+
+Time and duration constraints
+-----------------------------
+
+Clients and vehicles are equipped with time windows, and a number of other time and duration-related attributes.
+Together, these attributes can be used to model a rich set of constraints.
+Below, we explain the attributes, and how these can be used to enforce different constraints.
+
+Clients
+^^^^^^^
+
+In the case of clients modelled using the :class:`~pyvrp._pyvrp.Client` object, the time window indicates when service at the client may *begin*.
+Service is allowed to complete after the time window is already closed.
+A vehicle may arrive at a client before its time window opens, in which case it waits until the opening of the time window to begin service.
+Arriving late, that is, after the client time window closes, is not allowed in a feasible solution.
+Service takes a certain duration to complete, after which the vehicle is free to leave the client.
+The following figure explains this graphically.
+
+.. figure:: ../assets/images/duration-client.svg
+   :alt: Duration attributes of ``Client`` objects.
+   :figwidth: 100%
+
+.. hint::
+
+   You can model clients with multiple time windows using a mutually exclusive client group.
```

Comment:
I think we should (1) remove this hint and make our own notebook demonstrating this or (2) refer to https://github.com/PyVRP/PyVRP/discussions/673.

Reply:
We can also just remove it. The reference to the discussion seems unnecessary; I'd rather have a runnable example. But that can come later.